web/packages/teleport/src/components/LabelsInput/LabelsInput.tsx

```diff
@@ -273,3 +288,40 @@ const Legend = styled.legend`
   padding: 0;
   ${props => props.theme.typography.body3}
 `;
+
+const LabelTable = styled.table`
```

Comment:
> This PR changes the internal structure of `LabelsInput` from a bunch of boxes to a regular table, which is also more accessible.

Does it mean that tables are inherently more accessible in this scenario? I didn't know about this, but I wouldn't mind reading up on it, do you have any good resources on this?

Reply:
Yes, they are better. Though I don't have any actual reading material that I could recommend, let me jus prove my case with [a demo of how Apple's own VoiceOver reads our UI](https://goteleport.zoom.us/clips/share/UxJu1QxqSIOpsrU_JHYh9g). It would be even better to use a grid role for this, as this is an interactive table, but that would, by convention, require us to implement a different keyboard navigation scheme, so I didn't do it. Table, however, was a low-hanging fruit here, since I already needed to rebuild the layout anyway.

Reply:
Thanks for the recording!